tests/conftest.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+import pytest
+
+
+@pytest.fixture
+def height_samples_with_bounds():
+    """Output a vector of values simulating height samples with value bounds"""
+    n = 1000
+    height_min = 150
+    height_max = 200
+    height = np.random.uniform(height_min, height_max, n)
+    return height, height_min, height_max
+
+
+@pytest.fixture
+def weight_samples_with_bounds():
+    """Output a vector of values simulating weight samples with value bounds"""
+    n = 1000
+    height = np.random.uniform(150, 200, n)
```

Comment:
NIT: `height_samples_with_bounds` and `weight_samples_with_bounds` values are dependant but we don't have it in the way you implemented the functions, ie if we change values in one and not the other then they won't be validate the covariance test (if I am correct). Can't you use a function (not a fixture one) to distribute the height values in both?

Reply:
`n` should be mutualised, otherwise the test will indeed fail, but actually we don't need height and weight to be correlated, the correlation or non-correlation should only be approximately equal with and without DP, this is what the test checks.
If you agree with this, I will just factorize `n`